src/Storage/Commands.Storage/File/Cmdlet/GetAzureStorageShare.cs

```diff
@@ -28,7 +29,7 @@ public class GetAzureStorageShare : AzureStorageFileCmdletBase
             Position = 0,
             Mandatory = true,
             ParameterSetName = Constants.SpecificParameterSetName,
-            HelpMessage = "Name of the file share to be listed.")]
+            HelpMessage = "Name of the file share to be gotten.")]
```

Comment:
@blueww I think we should change this to "Name of the file share to be received"